Kitodo/src/test/java/org/kitodo/MockDatabase.java

```diff
@@ -17,20 +17,19 @@
 import static org.kitodo.data.database.beans.Batch.Type.SERIAL;
 
 import de.sub.goobi.helper.Helper;
-
```

Comment:
Removing empty lines between the imports is not needed.

Reply:
This is done automatically when organizing imports with Eclipse because we have package grouping disabled. If I would undo it, it would happen again the next time one line is changed in the file.

Reply:
I think this could never be unified between different IDE's / tools. I don't like it and would prefer a empty line between different package names but I would accept it.